pkg/lint/lintersdb/manager.go

```diff
@@ -298,6 +298,12 @@ func (m Manager) GetAllSupportedLinterConfigs() []*linter.Config {
 			WithLoadForGoAnalysis().
 			WithURL("https://github.com/kkHAIKE/contextcheck"),
 
+		linter.NewConfig(golinters.NewCopyLoopVar()).
+			WithSince("v1.57.0").
+			WithLoadForGoAnalysis().
```

Comment:
```suggestion
```

Reply:
I fixed it.
https://github.com/golangci/golangci-lint/pull/4382/commits/bbefb6091d23c06e88897fd9fa0cf2926a0b8f9d